package.json

```diff
@@ -11,7 +11,8 @@
     "lodash": "^4.17.21",
     "nanoid": "^3.1.25",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "svg-to-excalidraw": "file:./svg-to-excalidraw-0.0.1.tgz"
```

Comment:
This is with the patch reported at https://github.com/excalidraw/svg-to-excalidraw/issues/87